src/components/card-menu/card.css

```
@@ -8,10 +8,11 @@
   justify-content: center;
   flex-shrink: 0;
 
-  transition: 
-    transform .2s ease-in-out,
-    color .2s ease-in-out,
-    background-color .2s ease-in-out;
+  text-align: center;
+  white-space: normal;
+  word-wrap: break-word;
+
+  transition: transform 0.2s ease-in-out;
```

Percebi depois só mas aqui vc tá sobrescrevendo uma transição q eu tô usando, então se puder corrigir pq senão eu vou ter q literalmente reescrever as linhas q foram substituídas dnv.
```suggestion
  text-align: center;
  white-space: normal;
  word-wrap: break-word;

  transition:
    transform .2s ease-in-out,
    color .2s ease-in-out,
    background-color .2s ease-in-out;
```